backend/compact-connect/tests/smoke/military_affiliation_smoke_tests.py

```diff
@@ -0,0 +1,131 @@
+# ruff: noqa: S101  we use asserts for smoke testing
+import os
+import sys
+import time
+from datetime import UTC, datetime
+
+import requests
+
+# This script is used to test the military affiliations upload flow of the Compact Connect API.
+# It is currently configured to run locally against a sandbox environment. To run this script, you must
+# set the following constant values to match your environment
+# This should include the https:// prefix  (ex. https://sandbox.compactconnect.org)
+API_URL = os.environ['API_URL']
+# The cognito ID token issued when the user is authenticated.
+TEST_USER_COGNITO_ID_TOKEN = os.environ['TEST_USER_ID_TOKEN']
```

Comment:
How does one get this token if they want to run these smoke tests? Can we spell this out a bit more (e.g., the recommended workflow for getting a token)?

Reply:
I've added a comment walking through the simplest steps to get the id token.